boudicca.events/eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/AlpenvereinCollector.kt

```diff
@@ -12,9 +12,13 @@ import base.boudicca.model.EventCategory
 import base.boudicca.model.structured.StructuredEvent
 import org.jsoup.Jsoup
 import org.jsoup.nodes.Document
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.stereotype.Component
 import java.net.URI
 import java.util.concurrent.TimeUnit
 
+@Component
```

Comment:
it probably would be nice to create our own meta-annotation @EventCollector or something, we could use this to somehow configure the eventcollector if the need arises as well